tfhe/src/shortint/server_key/mod.rs

```diff
@@ -54,14 +57,77 @@ impl Display for CheckError {
 
 impl std::error::Error for CheckError {}
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub enum ShortintBootstrappingKeyBase {
```

Comment:
I'm going to be annoying, but can we also rename this to `ShortintBootstrappingKey` ? 😅

Reply:
done